oak_idl/src/lib.rs

```diff
@@ -14,14 +14,10 @@
 // limitations under the License.
 //
 
-//! A Rust-based IDL implemented via `macro_rules`. It is inspired by
-//! <https://github.com/google/tarpc> and
-//! <https://fuchsia.dev/fuchsia-src/reference/fidl/language/wire-format>.
-//!
-//! For each service definition, it generates both a client stub and a service trait, as well as a
-//! method to dispatch invocations to a service implementation.
-
 use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
```

Comment:
This should probable use `core::fmt::Debug` and have a `#[no_std]` attribute in this file for compatibility with the bare-metal runtime.

Reply:
Done.